docs/site/content/docs/latest/designs/package-process.md

```diff
@@ -589,7 +589,7 @@ packages:
 
 There is a makefile task, `generate-package-repo`, that generates the package repository from this file. `kapp-controller`
 currently expects the package repositories to be in the format of an imgpkgBundle. This task will generate that bundle.
-When the task is executed, `make generate-package-repo CHANNEL=main`, the following steps are performed:
+When the task is executed, `make generate-package-repo CHANNEL=main tAG=0.11.0`, the following steps are performed:
```

Comment:
is `tAG` a typo?

Reply:
😞 yes, would mind making a suggestion to `TAG` so  that I can easily accept it?